project.clj

```diff
@@ -2,12 +2,28 @@
   :description "A HTTP route matching library"
   :url "https://github.com/weavejester/clout"
   :license {:name "Eclipse Public License"
-            :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]
-                 [instaparse "1.4.0" :exclusions [org.clojure/clojure]]]
+            :url  "http://www.eclipse.org/legal/epl-v10.html"}
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "1.7.228" :scope "provided"]
+                 [instaparse "1.4.8" :exclusions [org.clojure/clojure]]]
+  :plugins [[lein-doo "0.1.7"]
+            [lein-cljsbuild "1.1.4"]]
   :profiles
-  {:dev {:jvm-opts ^:replace []
+  {:dev {:jvm-opts     ^:replace []
          :dependencies [[ring/ring-mock "0.2.0"]
                         [criterium "0.4.2"]]}
-   :1.6 {:dependencies [[org.clojure/clojure "1.6.0"]]}
-   :1.7 {:dependencies [[org.clojure/clojure "1.7.0-beta2"]]}})
+   :1.7 {:dependencies [[org.clojure/clojure "1.7.0"]
+                        [org.clojure/clojurescript "1.7.228"]]}
+   :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]
+                        [org.clojure/clojurescript "1.8.51"]]}
+   :1.9 {:dependencies [[org.clojure/clojure "1.9.0"]
+                        [org.clojure/clojurescript "1.9.946"]]}}
+  :cljsbuild {:builds
```

Comment:
As a minor point, could you ensure the profiles come last? That makes it easier to see which keys are "base" and which are profiles.